packages/module/src/ChatbotConversationHistoryNav/LoadingState.tsx

```diff
@@ -0,0 +1,39 @@
+import { Skeleton } from '@patternfly/react-core';
+import React from 'react';
+
+export interface LoadingStateProps {
+  'aria-label'?: string;
```

Comment:
The prop name should be `screenreaderText` to match what the text will be used for

Reply:
All set; thank you!